services/api/mw2pdf/src/classes/MediaWikiSession.ts

```diff
@@ -162,7 +166,8 @@ export class MediaWikiSession {
           title: 'No Title',
           output: null,
           workDirectory: options.workDirectory,
-          makeTitlePage: MwPdfMakeTitlePage
+          makeTitlePage: MwPdfMakeTitlePage,
+          pageSize: options.pageSize,
```

Comment:
I've been scouring this whole PR for something to complain about, and I can't find a single dang thing.

Here you even take a list that didn't end with a trailing comma and, in the course of adding a new element, also add the trailing comma.  If anything, that seems like an improvement.  So now I'm down one -- if I find anything to complain about after this point, it'll just bring me back to zero.

Thanks a lot.  Sheesh.